shell.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import <nixpkgs> {} }:
+with pkgs;
+
+let
```

Comment:
What is the purpose of an empty `let / in` statement? Is it just by convention for potential future values?

Reply:
Nah that's merely just a leftover. I used to have stuff in there which disappeared over time. I will remove this.